package.json

```diff
@@ -20,5 +20,8 @@
   "resolutions": {
     "broccoli-asset-rewrite@^2.0.0": "patch:broccoli-asset-rewrite@npm%3A2.0.0#./.yarn/patches/broccoli-asset-rewrite-npm-2.0.0-c4ce42084a.patch",
     "@glint/environment-ember-loose": "patch:@glint/environment-ember-loose@npm%3A1.4.0#~/.yarn/patches/@glint-environment-ember-loose-npm-1.4.0-31c2f31bcb.patch"
+  },
+  "dependencies": {
+    "clipboard-polyfill": "^4.1.0"
```

Comment:
question: Did you mean to put this here?

Reply:
nope! will remove.